lib/rubocop/cop/style/if_with_boolean_literal_branches.rb

```diff
@@ -8,6 +8,20 @@ module Style
       # The conditions to be checked are comparison methods, predicate methods, and double negative.
       # `nonzero?` method is allowed by default.
       # These are customizable with `AllowedMethods` option.
+      # Multiple `elsif`s with booleans in branches are allowed for readability.
```

Comment:
Perhaps it'd be better to mention that this cop targets only `if`s with a single `else` branch. This seems clearer to me.

Reply:
Thank you! I've tweaked the text to "This cop targets only `if`s with a single `elsif` or `else` branch".